crypto/src/main/kotlin/web5/sdk/crypto/Main.kt

```diff
@@ -0,0 +1,3 @@
+package web5.sdk.crypto
+
+public fun main() {}
```

Comment:
this file is needed?

Reply:
nope we can ditch it!

Reply:
done: https://github.com/TBD54566975/web5-sdk-kotlin/commit/69b8c0e72a8e8ea60737c1ce5a7411546dc8b61f